src/components/Sidebar.js

```diff
@@ -0,0 +1,82 @@
+import React, { Fragment } from 'react';
+import { Link } from 'gatsby';
+
+// Toggles sidebar menus, only one may be open at a time
+function toggleMenu(id) {
+  const element = document.getElementById(id);
+
+  const scanners = document.getElementById('Scanners');
+  const persistenceProviders = document.getElementById('Persistence Providers');
+  const hooks = document.getElementById('Hooks');
+
+  if (element !== scanners) {
+    scanners.style.display = 'none';
+  }
+  if (element !== persistenceProviders) {
+    persistenceProviders.style.display = 'none';
+  }
+  if (element !== hooks) {
+    hooks.style.display = 'none';
+  }
+
+  element.style.display === 'block'
```

Comment:
I don't think this is a good use of a ternary operator.
Doing the assignment **inside** the ternary block is really unexpected and confused me.
Would prefer a simple if statement here

Reply:
Good catch. Since it is a simple, conditional value assignment, the ternary operator should be perfectly readable and preferable in use. But I did not use it correctly, it should look more like:
`var myVar = condition ? optionA : optionB`
instead of the assignment inside each case.  Will be corrected asap ^^